### PR TITLE
Update latexify tests

### PR DESCRIPTION
**Pull request by adamslc**
xref JuliaSymbolics/Symbolics.jl#183.

### CODE REVIEW

**Review by YingboMa — APPROVED** [review]
MWTP

**Comment by adamslc** [comment]
I think that this should make CI pass again, but not necessarily local tests.

The issue is that the new latexify code in `Symbolics` iterates over the dict in any `Mul`s, which is undefined behavior. Working on a fix now.

File: test/runtests.jl [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
-@safetestset "Lowering Integration Test" begin include("lowering_solving.jl") end
-@safetestset "Test Big System Usage" begin include("bigsystem.jl") end
-@safetestset "Depdendency Graph Test" begin include("dep_graphs.jl") end
-@safetestset "Function Registration Test" begin include("function_registration.jl") end
-@safetestset "Precompiled Modules Test" begin include("precompile_test.jl") end
-@testset "Distributed Test" begin include("distributed.jl") end
-@safetestset "Variable Utils Test" begin include("variable_utils.jl") end
-println("Last test requires gcc available in the path!")
-@safetestset "C Compilation Test" begin include("ccompile.jl") end
+# @safetestset "Varialbe scope tests" begin include("variable_scope.jl") end
```
Comment (YingboMa) [inline test/runtests.jl:3]: Oof

Comment (adamslc) [reply test/runtests.jl:3]: Damm, sorry I let that slip in.

Comment (YingboMa) [reply test/runtests.jl:3]: Np, I fixed it in #973

**Pull request merged**